docs/topics/vs-code-extension-interface.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id="vs-code-extension-interface_{context}"]
+= {ProductShortName} extension interface
+
+The interface of the {ProductName} ({ProductShortName}) extension is designed to make it easier for you to find information and perform actions:
+
+* In the left pane, you can see icons representing the applications analyzed by MTA, the rulesets used, and the issues discovered by the analysis. You can also click the report icon to open the MTA report in your browser.
+
+* In the upper right pane, you can configure an analysis.
+
+* In the lower right pane, you can see the settings of the configuration, including source, target, and advanced options. During an analysis, you can track its progress in this pane. When the analysis is completed, you can click a button to open the MTA report, which describes any issues you need to address before you migrate or modernize your application. For more information, see link:{ProductDocUserGuideURL}/index#review_reports_cli-guide[{ProductShortName} report] in the _{UserCLIBookName}_.
```

Comment:
"During an analysis, you can track its progress" > "You can view the progress of an analysis in this pane."

"you can click a button" Does the button have a label? If so, be specific.